core/src/types.ts

```diff
@@ -356,3 +356,18 @@ export interface IErrorMsgData {
   name: string
   stack: string
 }
+
+// Bisontry Interface
```

Comment:
@jay-bisonai Shouldn't the interface be imported from `Bisontry`?

Reply:
Thanks for letting me know, I agree we should import from `Bisontry`.
But this interface hasn't reached the point where other repositories can access yet. I will make a plan with infra-team to change to make it accessible from another repository.

Reply:
@martinkersner Thanks for waitting! we decide postponed adding to send error-report to Bisontry in Orakl/monitor.
This feature will be added after we modify Bisontry interface so that it can be imported from another repository.

Also, this code moved to another branch.
https://github.com/Bisonai/orakl/tree/i-801/feat/bisontry-error-report